### ISSUE TITLE
NVDA announces the Desktop window as program manager and fails to announce desktop names

### ISSUE THREAD

**Issue by amirsol81** [nvaccess/nvda#19181]
### Brief summary

NVDA, 64-Bit alpha or official 2025.3.1, portable and installed, fails to properly report the title of the Desktop window and its count, like "Desktop 2," when the focus is on the Desktop. It's oddly reported as Program Manager. JAWS and Narrator work fine, and JAWS even reports the Desktop count properly.

### Steps to reproduce

Simply make sure the focus is on the Desktop window by pressing WINDOWS+M or Windows+D, and press NVDA+T to check the window title.

### Actual behavior

NVDA says Program Manager.

### Expected behavior

NVDA should report both the title as "Desktop" and announce the Desktop count, like "Desktop 1." Windows 11 supports multiple Desktops.

### NVDA type

installer/launcher copy, installed copy

### NVDA version

All 64-Bit alphas (like 53258), and the official 2025.3.1 release.

### Have you tried any other versions of NVDA? If so, please report their behaviors.

Yes - the same. 64-Bit alphas and official 2025.3.1. Desktop is always reported as Program Manager, and no Desktop count is reported either.

### Windows version

Newer Windows 11 24H2 Pro builds, and now Windows 11 25H2 build 26200.7019

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by Qchristensen** [comment]
Just a minor clarification - "Desktop 1", "Desktop 2" etc are not the count of the desktops, but the default name - in the same way as the first blank file you create in Word is "Document 1", the next is "Document 2" and so on.  You can name desktops (Press windows+tab then press tab to the list of windows, arrow between them and press F2 or open the context menu and select "rename").

And also a workaround for now - If you press windows+tab to open the task switching window, then press tab, it reports the current desktop if you press tab, the focus moves to the list of desktops with the focus on the first one.  NVDA will read the name of the desktop, it's count ("1 of 3" for instance) and it will say "Not selected" if this is NOT the active desktop.  If this is the active desktop, it only reads the name and count.  Press escape to close.

Another workaround is that if you press windows+control+right arrow / Windows+control+left arrow it switches between desktops and does report the name - You could do this to ensure you are on the desired desktop.

**Comment by amirsol81** [comment]
@Qchristensen This issue continues with Windows 11 25H2, NVDA 2026.2, and all alphas.

**Comment by Qchristensen** [comment]
@amirsol81 Just to check, is this the names of the desktops not being announced correctly as you press Windows+control+right or left arrow to move between virtual desktops?

Or is it the desktop not being reported correctly when you press Windows+d to minimise everything and move to the desktop?

In both cases everything seems to be working here and being reported as expected with:

NVDA 2026.2 on
Windows 11 (64-bit) Version: 25H2, Build: 26200.9278 
If you press NVDA+q then down arrow to "restart with add-ons disabled and debug logging enabled" does it still happen when NVDA restarts?

**Comment by Qchristensen** [comment]
Quick clarification from Mastodon conversation on this issue - I can replicate that while my above comment stands, if I press NVDA+t to read the title bar, it does read as program manager.

**Comment by geoffshang** [comment]
On Mon, 31 Aug 2026, Quentin Christensen wrote:

> Quick clarification from Mastodon conversation on this issue - I can replicate that while my above comment stands, if I press NVDA+t to read the title bar, it does read as program manager.

Hasn't it always done this?

**Comment by amirsol81** [comment]
@geoffshang No. It started with later updates made to Windows 11 24H2. It's a relatively long-standing issue affecting NVDA's title reporting with NVDA+T, while JAWS and Narrator aren't affected.
